Dokumentationen/architecture/architecture_notebook.adoc

```diff
@@ -17,52 +24,275 @@ Dieses Dokument beschreibt die Philosophie, Entscheidungen, Nebenbedingungen, Be
 
 //Formulieren Sie eine Reihe von Zielen, die die Architektur in ihrer Struktur und ihrem Verhalten erfüllen muss. Identifizieren Sie kritische Fragen, die von der Architektur adressiert werden müssen, z.B. besondere Hardware-Abhängigkeiten, die vom Rest des Systems isoliert werden sollten oder Sicherstellung der Funktionsfähigkeit unter besonderen Bedingungen (z.B. Offline-Nutzung).
 
+Die Architektur muss eine Webanwendung zum Erstellen und Verwalten von Buchungen von Experiementen für Vorlesungen verwirklichen. Diese basiert auf einer bereits bestehenden Anwendung und ist somit als Weiterentwicklung zu betrachten. Aufgrund dessen muss die Architektur auf eben diese Weiterentwicklung bezogen sein und weiterhin durch Übersichtlichkeit, Eindeutigkeit und Verständlichkeit für eine hohe Nutzerfreundlichkeit sorgen.  
+
+Weitere Ziele, die die Architektur in Struktur und Verhalten erfüllen muss, sind: 
 
-== Annahmen und Abhängigkeiten
+* Gewährleistung einer hohen Nutzungsdauer - das System soll über einen langen Zeithorizont genutzt werden und muss für eine variierende Nutzergruppe ausgelegt sein
```

Comment:
ist Zeithorizont ein Wort?

Reply:
ja! sogar eins, dass in diesem Kotext durchaus passt :D